pkg/cmd/openshift-sdn-node/proxy.go

```diff
@@ -81,15 +82,19 @@ func (sdn *openShiftSDN) wrapProxy(s *ProxyServer, waitChan chan<- bool) error {
 		unidlingBroadcaster := record.NewBroadcaster()
 		unidlingBroadcaster.StartRecordingToSink(&corev1client.EventSinkImpl{Interface: sdn.informers.kubeClient.CoreV1().Events("")})
 		unidlingRecorder := unidlingBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: "kube-proxy", Host: sdn.nodeName})
-
+		// Starting from Kube 1.26, UDPIdleTimeout is no longer part of
+		// KubeProxyConfiguration, along with everything related to userspace proxy.
+		// Let's use here the default value it had in 1.25:
+		// https://github.com/openshift/kubernetes/blob/sdn-4.12-kubernetes-1.25.4/pkg/proxy/apis/config/v1alpha1/defaults.go#L72
```

Comment:
Comments should be written in a way that will make sense to readers in the future. This comment is written like it assumes that the reader is expecting `proxyConfig.UDPIdleTimeout` to exist, and so we need to explain why it doesn't. But future readers aren't going to expect that...

This doesn't need a very big explanation. Just say that it used to be configurable, but now it's not, so we use the old default value.